### PR TITLE
Fix path to golint for travis

### PR DESCRIPTION
**Pull request by pushrax**
Needed for CI to pass.

### CODE REVIEW

File: cmd/chihaya/config.go [hunk 1]
```diff
@@ -10,14 +10,10 @@ import (
 	"github.com/chihaya/chihaya/frontend/http"
 	"github.com/chihaya/chihaya/frontend/udp"
 	"github.com/chihaya/chihaya/middleware"
-
-	// Imported to register as middleware drivers.
```
Comment (mrd0ll4r) [inline cmd/chihaya/config.go]: Does this not pass through golint or goimports anymore? That'd be sad, I think it's quite useful here tbh

Comment (pushrax) [reply cmd/chihaya/config.go]: That's correct. I think it's obvious enough from the context, but the other options are:

- put the comment on the end of one of the import lines
- open new `import()` sections

Comment (mrd0ll4r) [reply cmd/chihaya/config.go]: I think we should add more import blocks to keep the comments.

File: cmd/chihaya/config.go [hunk 2]
```diff
@@ -10,14 +10,16 @@ import (
...
+import (
+	// Imports to register middleware drivers.
 	_ "github.com/chihaya/chihaya/middleware/clientapproval"
 	_ "github.com/chihaya/chihaya/middleware/jwt"
 	_ "github.com/chihaya/chihaya/middleware/torrentapproval"
 	_ "github.com/chihaya/chihaya/middleware/varinterval"
-
-	// Imported to register as storage drivers.
+)
+import (
```
Comment (jzelinskie) [inline cmd/chihaya/config.go]: Does gofmt not add new lines between import blocks? If it's true, then LGTM else, I'd prefer them to be here, it looks strange w/o them.

**Pull request merged**